core/src/main/java/org/pentaho/platform/util/Emailer.java

```diff
@@ -65,23 +65,23 @@
   public Emailer() {
   }
 
-  public void setTo( String to ) {
-    to = to.replaceAll( ";", "," );
+  void setTo( String to ) {
```

Comment:
@cravobranco, `removing` the public access modifier makes these methods invisible outside of the package and will break code using these methods from other packages. Please update the PR with the `public` access modifier.

See: https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html

Reply:
Tightening the access control as much as possible and avoiding to broaden it without reason is a basic Java OO rule - it enforces the Encapsulation property of this language.

Also, from the link you shared:
"
- Use the most restrictive access level that makes sense for a particular member. Use private unless you have a good reason not to.
- Avoid public fields except for constants. (Many of the examples in the tutorial use public fields. This may help to illustrate some points concisely, but is not recommended for production code.) Public fields tend to link you to a particular implementation and limit your flexibility in changing your code.
"